packages/entity-browser/src/components/SearchForm/SearchForm.js

```diff
@@ -19,25 +19,44 @@ const SearchForm = props => {
     })
   }
 
+  const showField = formDefinition => {
```

Comment:
i would just pass the fieldName (i think formDefiniton is wrong either way since its a field definitioN)